lib/Interpreter/CIFactory.cpp

```diff
@@ -316,14 +316,9 @@ namespace {
       }
 
     #if defined(__GLIBCXX__)
-      // Avoid '__float128 is not supported on this target' errors
```

Comment:
Please don't remove this comment, the original `sArguments.addArgument("-std=c++11")` was/is properly fixing the **__float128** problem.  The issue is just that it doesn't work for `-std=c++1z`